data/hails.txt

```diff
@@ -1246,6 +1246,107 @@ phrase "hostile quarg"
 		"You have only a short time to learn to live together in peace."
 		"You must live in peace... or leave the galaxy."
 
+
+phrase "hostile bounty hunter"
+	word
+		"You're going to make"
+	word
+		" me rich!"
+		" me filthy rich!"
+		" me a rich man!"
+		" me a rich woman!"
+		" me a lot of money!"
+		" me comfortable retirement!"
+		" my bank account happy!"
+		" for a nice bit of target practice!"
+
+phrase "hostile bounty hunter"
+	word
+		"I'm getting paid"
+		"I'm being paid"
+		"They paid me"
+		"My client paid me"
+	word
+		" handsomely"
+		" a lot of money"
+		" a huge amount of money"
+		" an enormous sum"
+		" an enormous sum of money"
+		" quite a bit"
+		" quite a bit of money"
+	word
+		" to"
+	word
+		" kill you"
+		" take you out"
+		" eliminate you"
+		" vaporize you"
+		" turn you into space debris"
+		" terminate you"
+		" destroy you"
+		" end your life"
+		" get rid of you"
+		" make you disappear"
+		" make you go away"
+	word
+		"."
+		"!"
+
+phrase "hostile bounty hunter"
+	word
+		"I have you now"
+		"You're mine"
+		"You will be destroyed"
+	word
+		","
+		", you"
+		"! Die,"
+		"! Prepare to die,"
+	word
+		" scum!"
+		" worm!"
+		" maggot!"
+		" filth!"
+		" miscreant!"
+		" fool!"
+		" poor fool!"
+		" unlucky sap!"
+
+phrase "hostile bounty hunter"
```

Comment:
This whole block feels really weird to me; it sounds like something you would say to your wingman, not transmit to your target as is the case here. That would be like saying "I'm about to kill you" before shooting them with a gun.

I'm fine with the other ones, but I'd recommend this block be deleted.